File: lcm.c
#include <stdio.h>

int main()
{
    int m,n,LCM;
    printf("Enter a two numbers: ");
    scanf("%d %d",&m,&n);
    
    LCM = (m>n)? m:n;

    while(1)
    {
        if(LCM%m==0 && LCM%n==0)
        {
            printf("LCM of %d and %d is = %d \n",m,n,LCM);
            break;
        }
        LCM++;
    }

    return 0;
}

File: Lcm.java
import java.util.Scanner;

public class lcm {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter number 1 : ");
		int no1 = sc.nextInt();
		System.out.println("Enter number 2 : ");
		int no2 = sc.nextInt();
		
		int big = no1>no2?no1:no2;
		
		while(true)
		{
			if(big%no1==0 && big%no2==0)
			{
				System.out.println(big);
				break;
			}
			big++;
		}
	}

}
